### PR TITLE
Alterations to ggplot2 lesson by Lindsay

### PR DESCRIPTION
**Pull request by lcoome**
Added new lesson, figures, interative html plot, R code

### CODE REVIEW

File: lessons/r/ggplot2/Lindsayggplot2.Rmd [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+always_allow_html: yes
+layout: page
+title: 'Plotting in R: ggplot2'
+visible: false
+tags:
+  - r
+  - intermediate
+output:
+  md_document:
+    variant: markdown_github
```
Comment (lwjohnst86) [inline lessons/r/ggplot2/Lindsayggplot2.Rmd]: These three lines can be changed to just `output: github_document`

File: lessons/r/ggplot2/Lindsayggplot2.Rmd [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+---
+
+# INTRODUCTION TO GGPLOT2
+* ##### Author: Lindsay Coome
+* ##### Lesson Topic: Plotting in R using ggplot2
```
Comment (lwjohnst86) [inline lessons/r/ggplot2/Lindsayggplot2.Rmd]: Please remove the `####` from both lines

Comment (lwjohnst86) [reply lessons/r/ggplot2/Lindsayggplot2.Rmd]: Also include the link to the github repo material (https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/r/ggplot2)

File: lessons/r/ggplot2/Lindsayggplot2.Rmd [hunk 3]
```diff
@@ -0,0 +1,152 @@
+---
```
Comment (lwjohnst86) [inline lessons/r/ggplot2/Lindsayggplot2.Rmd]: Rename this file to be `lesson.Rmd`. You can write over Sarah's lesson file.

File: lessons/r/ggplot2/Lindsayggplot2.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
+INTRODUCTION TO GGPLOT2
```
Comment (lwjohnst86) [inline lessons/r/ggplot2/Lindsayggplot2.md]: This file will be changed after you rename the other file to `lesson.Rmd`.

File: lessons/r/ggplot2/Lindsayggplot2.Rmd [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+title: 'Plotting in R: ggplot2'
+visible: false
+tags:
+  - r
+  - intermediate
+output:
+  md_document:
+    variant: markdown_github
+---
+
```
Comment (lwjohnst86) [inline lessons/r/ggplot2/Lindsayggplot2.Rmd]: You can add this code chunk above:

    ```{r, echo=FALSE}
    knitr::opt_chunk$set(fig.path = 'figure/')
    ```

Then re-knit the file.

File: lessons/r/ggplot2/Lindsayggplot2_files/figure-markdown_github/Imagefiles.Rmd [hunk 6]
```diff
@@ -0,0 +1 @@
+##Contains all image files
```
Comment (lwjohnst86) [inline lessons/r/ggplot2/Lindsayggplot2_files/figure-markdown_github/Imagefiles.Rmd]: These files will be renamed after including the above code chunk (`knitr::opt_chunk$set(fig.path = 'figure/')`)

Comment (lwjohnst86) [reply lessons/r/ggplot2/Lindsayggplot2_files/figure-markdown_github/Imagefiles.Rmd]: You may likely need to delete all of these figure files.

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]

File: lessons/r/ggplot2/Lindsayggplot2.Rmd [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+# INTRODUCTION TO GGPLOT2
+* ##### Author: Lindsay Coome
+* ##### Lesson Topic: Plotting in R using ggplot2
+
+## WHAT IS GGPLOT?
+ggplot2 is meant to be an implementation of the Grammar of Graphics, hence gg-plot. The basic notion is that there is a grammar to the composition of graphical components in statistical graphics, and by direcly controlling that grammar, you can generate a large set of carefully constructed graphics tailored to your particular needs. Each component is added to the plot as a layer.
```
Comment (QuLogic) [inline lessons/r/ggplot2/Lindsayggplot2.Rmd]: direcly -> directly

File: lessons/r/ggplot2/Lindsayggplot2.Rmd [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+```
+
+## MORE RESOURCES ON GGPLOT2
+[ggplot2 documentation](http://had.co.nz/ggplot2/)
+
+[Book by Hadley Wickham](https://www.amazon.com/ggplot2-Elegant-Graphics-Data-Analysis/dp/0387981403)
+
+[ggplot2 cheat sheet](https://www.rstudio.com/wp-content/uploads/2015/03/ggplot2-cheatsheet.pdf)
+
+[r graph gallery for inspiration (not just limited to ggplot2 graphs)](http://www.r-graph-gallery.com/all-graphs/)
```
Comment (QuLogic) [inline lessons/r/ggplot2/Lindsayggplot2.Rmd]: https

**Comment by QuLogic** [comment]
32 messages? Wow, working on the GitHub web interface sure is annoying...

**Comment by lcoome** [comment]
Agreed
________________________________

**Pull request closed without merging**